scripts/bazel/build-packages.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+
+# Exit if any of the builds fails
+set -e
+
+# Source directories for packages to build with bazel and copy into dist/
+packages=(cdk lib cdk-experimental material-experimental material-moment-adapter)
+
+# The bazel-bin directory where the bazel output is written
+bazel_bin="$(bazel info bazel-bin)"
+
+# The dist/ directory where we want to copy the final result. Clear it out to avoid
+# any artifacts from earlier builds being retained.
+packages_dist="./dist/bazel-packages"
+rm -rf ${packages_dist}
```

Comment:
Does this actually do what you want? you want bazel clean but I think you'll just create another symlink into the output directory which is read-only

Reply:
I don't follow what you mean. I'm copying the the output from bazel-bin, copying it to `./dist` and then modifying it (no symlink). I wipe out the directory in `./dist` to make sure something from an earlier bazel build doesn't stick around.